Core/Core/Planner/CalendarViewController.swift

```diff
@@ -151,7 +151,7 @@ class CalendarViewController: UIViewController {
 
     func updateExpanded() {
         daysHeight.constant = isExpanded ? days.maxHeight : days.minHeight
-        dropdownView.transform = CGAffineTransform(rotationAngle: isExpanded ? -.pi : 0)
+        dropdownView.transform = CGAffineTransform(rotationAngle: isExpanded ? .pi : 180 * .pi)
```

Comment:
I think this is in radians, so there shouldn't be a 180

Reply:
I think pi / 180 is how you convert to radians, we essentially want it to rotate from 0 to pi, so this rotates it the same direction (by multiplying) as the `setHeight` method.  I'm not the sharpest at geometry so I could be wrong here.  Happy to change if you have a suggested value or equation.

Reply:
I'm not quite sure what the intended behavior is, so I can't say for sure what the correct number would be. 2π would be a complete rotation, and 180π is 90 rotations, which it probably just wraps to be the same as 2π.